enterprise/management/commands/create_enterprise_course_enrollments.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+"""
+Django management command for creating EnterpriseCourseEnrollment records.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import logging
+
+from edx_rest_api_client.exceptions import HttpClientError
+from requests.exceptions import ConnectionError, Timeout
```

Comment:
In order to avoid the quality error and reduce the # of exceptions used here, we can just import `RequestException`, the parent of these both.